can-stache-bindings.js

```diff
@@ -468,25 +468,66 @@ var behaviors = {
 					expr = new expression.Call(expr, defaultArgs, {} );
 				}
 
-				// make a scope with these things just under
-				var localScope = data.scope.add({
-					"@element": el,
-					"@event": ev,
-					"@viewModel": viewModel,
+				var templateContext = data.scope.getTemplateContext()._context;
+
+				function makeWarning(prefix, property, value){
+					return function(){
+						//!steal-remove-start
+						var filename = canReflect.getKeyValue(templateContext, 'filename');
+						var lineNumber = canReflect.getKeyValue(templateContext, 'lineNumber');
+						dev.warn(
+							(filename ? filename + ': ' : '') +
+							(lineNumber ? lineNumber + ': ' : '') +
+							prefix + property + " is deprecated. Use scope." + property + " instead."
+						);
+						//!steal-remove-end
+						return value;
+					};
+				}
+
+				canReflect.setKeyValue(templateContext, 'scope.element', el);
+				canReflect.setKeyValue(templateContext, 'scope.event', ev);
+				canReflect.setKeyValue(templateContext, 'scope.viewModel', viewModel);
+				canReflect.setKeyValue(templateContext, 'scope.arguments', arguments);
+
+				var specialValues = {
 					"@scope": data.scope,
 					"@context": data.scope._context,
 
-					"%element": this,
 					"$element": types.wrapElement(el),
-					"%event": ev,
-					"%viewModel": viewModel,
 					"%scope": data.scope,
-					"%context": data.scope._context,
-					"%arguments": arguments
-				},{
-					notContext: true
+					"%context": data.scope._context
+				};
+
+				Object.defineProperties(specialValues, {
```

Comment:
Can you `steal-remove` this entire Object.defineProperties instead of just removing the dev warning?